src/cmd-buildextend-metal

```diff
@@ -136,6 +136,9 @@ fi
 mkdir -p tmp/anaconda
 
 for itype in ${BIOS:+metal-bios} ${UEFI:+metal-uefi} ${DASD:+metal-dasd}; do
+    if [ "${arch}" = "s390x" ] && [ "$itype" = "metal-bios" ]; then
+        itype="metal-zfcp"
+    fi
```

Comment:
should we allow "metal-zfcp" for other arches? it might be confusing for a non s390x user...although it is nothing but a rename

Reply:
`should we allow "metal-zfcp" for other arches?` : no.
My idea is to have the `rhcos-<version>-metal-zfcp.raw.gz` up in the mirror.openshift.com so that s390x people don't get confused.

To be honest, I don't know how make this prettier. I would go with whichever way you prefer.

Reply:
i think if someone tries `cosa buildextend-metal --zfcp` on non s390 arches we should just display an info message that it will generate a bios image